vm/runtime/src/lib.rs

```diff
@@ -211,9 +215,46 @@ pub trait Syscalls {
         Ok(())
     }
     /// Verifies a proof of spacetime.
-    fn verify_post(&self, _vi: &WindowPoStVerifyInfo) -> Result<(), Box<dyn StdError>> {
-        // TODO
-        todo!()
+    fn verify_post(&self, verify_info: &WindowPoStVerifyInfo) -> Result<(), Box<dyn StdError>> {
+        type ReplicaMapResult = Result<(SectorId, PublicReplicaInfo), String>;
```

Comment:
Hmm the map couldn't infer the type and I didn't want to verbose the associate type cause it would be hard to read . Maybe the naming could be better?

Reply:
fine the way it was, just seemed weird on first read to alias something only used once